src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const messageDiv = document.createElement('div');
+
+messageDiv.classList.add('message');
+
+function handler() {
+  const promise1 = Promise.resolve('Promise was resolved!');
+
+  promise1.then((result) => {
+    messageDiv.innerText = result;
+    document.body.appendChild(messageDiv);
+  });
```

Comment:
You're creating a Promise and immediately resolving it. This is not wrong, but in a real-world scenario, you'd use Promises for asynchronous operations. Consider simulating this with a setTimeout.